src/index.html

```diff
@@ -11,7 +11,166 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="get">
+      <fieldset class="border-field">
+        <legend>Personal information</legend> 
+        <label class="form-field">
+        Surname:
+          <input type="text" 
+          name="surname"
+          autocomplete="off"
+          >
```

Comment:
```suggestion
        <label class="form-field">
          Surname:
          <input 
            type="text" 
            name="surname"
            autocomplete="off"
          >
```
Текст вложен в label, у него тоже должен быть отступ
Теги форматирую таким образом